Makefile

```diff
@@ -58,7 +58,7 @@ JUNIT_FILE ?= /tmp/report.xml
 
 %.md:
 
-all: build
+all: build-kfctl
```

Comment:
Instead of doing so many changes, I think you can just
```
build-kfctl: build
```
Did I miss anything?

Reply:
Well the reason why I removed this `build` target is because it does the samething exactly as `build-kfctl` and it seems redundant. If we bring that back it will be the same....

Reply:
Understood, I was more thinking of this as a hotfix to unblock manifests presubmit test, so I prefered the change to be smaller (one line fix), but this also doesn't seem risky. If you think `build-kfctl` is a good name for this target for the long term, I don't have any objections.
/lgtm

Reply:
@Bobgy I checked the kubeflow/testing scripts and it seems that I have to submit another PR to change the binary name too, as currently the master branch makes the name of the binaries as `kfctl-<OS>-<ARCH>` instead of a simple `kfctl`...

Reply:
Shall we revert the original PR instead of forward fixing then?
You can decide

Reply:
Yes maybe revert is a better idea before we identified how this will break everything...I will try to submit another PR later to add multiarch support again then.